src/providers/webviewProvider/requestMethods.ts

```diff
@@ -0,0 +1,79 @@
+import { PlatformsEnum, WebViewProviderRequestEnums } from "types/index";
+
+export const requestMethods = {
```

Comment:
I think you should set the targetOrigin for each post message and you should accept the message just from the desired origins (you should whitelisting the origins in your client)